### ISSUE TITLE
Data types in API response

### ISSUE THREAD

**Issue by wy** [neo-ngd/Happynodes#22]
Current API's /nodes route returns strings unnecessarily:
```
 {
        "id": **"24"**,
        "hostname": "node2.sgp1.bridgeprotocol.io",
        "protocol": "http",
        "address": "http://node2.sgp1.bridgeprotocol.io",
        "validated_peers_counts": 15,
        "stability": **"100"**,
        "blockheight_score": **"100.00000000000000000000"**,
        "normalised_latency_score": 78.8694739341736,
        "validated_peers_counts_score": **"100.00000000000000000000"**,
        "health_score": 94.7173684835434,
        "latency": 0.422610521316528,
        "online": true,
        "locale": "sg"
      }
```

Please fix the bold above to Integer or Float whichever is applicable.

**Comment by wy** [comment]
See the ones marked with asterisks.
Please also review /nodes/:id which has a similar problem
And also /unconfirmed where node_count is a string.

**Comment by i25959341** [comment]
fixed by #28
